src/Scroll/ScrollView.tsx

```diff
@@ -223,7 +226,7 @@ export class ScrollView extends React.Component<ScrollView.Props> {
           this.scrollView = sv;
         }}
       >
-        {t.Function.is(children) ? children(this.scrollTo) : children}
+        {isFunction(children) ? children(this.scrollTo) : children}
```

Comment:
replace with `typeof children === "function"`?

Reply:
This change fails with a weird error:
```ts
This expression is not callable.
  Not all constituents of type '((scrollTo: ScrollTo) => Children) | (string & {}) | (number & {}) | (false & {}) | (true & {}) | (((scrollTo: ScrollTo) => Children) & string) | (((scrollTo: ScrollTo) => Children) & number) | ... 4 more ... | (((scrollTo: ScrollTo) => Children) & ReactPortal)' are callable.
    Type 'string & {}' has no call signatures.ts(2349)
```

Not sure how to fix without using an `any` cast really..